src/pybind/mgr/cephadm/module.py

```diff
@@ -93,6 +93,8 @@ def os_exit_noop(status: int) -> None:
 DEFAULT_IMAGE = 'quay.io/ceph/ceph'
 DEFAULT_PROMETHEUS_IMAGE = 'quay.io/prometheus/prometheus:v2.18.1'
 DEFAULT_NODE_EXPORTER_IMAGE = 'quay.io/prometheus/node-exporter:v0.18.1'
+DEFAULT_LOKI_IMAGE = 'docker.io/grafana/loki:2.4.0'
+DEFAULT_PROMTAIL_IMAGE = 'docker.io/grafana/promtail:2.4.0'
```

Comment:
do any images for these containers exist on quay? We're generally trying to move away from docker whenever it's possible to avoid the rate limiting (e.g. https://github.com/ceph/ceph/pull/44894). If this is the only place we can get them so be it, but if there are alternatives it would be preferred.

Reply:
I checked..but didn't find any